src/main/java/frc/robot/hardware/mechanisms/MechanismSimulation.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot.hardware.mechanisms;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public interface MechanismSimulation {
+
+	Rotation2d getRotorPosition();
+
+	Rotation2d getRotorVelocityAnglesPerSecond();
+
+	Rotation2d getSystemPosition();
+
+	Rotation2d getSystemVelocityAnglesPerSecond();
```

Comment:
If the file is called Mechanism simulation, why are these using the name "system"?

Reply:
in PR: names